sqs/templates/lambda_policy.json.tpl

```diff
@@ -0,0 +1,18 @@
+{
+  "Version": "2012-10-17",
+  "Id": "sns_topic_policy",
+  "Statement": [
+    {
+      "Sid": "sns_topic_statement",
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "SQS:SendMessage",
+      "Resource": "arn:aws:sqs:${region}:${account_id}:${sqs_name}",
+      "Condition": {
+        "ArnEquals": {
+          "aws:SourceArn": "arn:aws:lambda:${region}:${account_id}:*"
```

Comment:
Also, what's the reason for putting this in a Condition rather than the Principal? This might just be me getting confused about IAM policy syntax.

Reply:
Principal is either an account id or a user or a role, sort of who can access it. The conditions filter what is allowed to access the queue.